src/snowflake/connector/ocsp_snowflake.py

```diff
@@ -74,38 +74,50 @@ class OCSPResponseValidationResult(NamedTuple):
     validated: bool = False
 
 
-try:
-    OCSP_RESPONSE_VALIDATION_CACHE: SFDictFileCache[
-        tuple[bytes, bytes, bytes],
-        OCSPResponseValidationResult,
-    ] = SFDictFileCache(
-        entry_lifetime=constants.DAY_IN_SECONDS,
-        file_path={
-            "linux": os.path.join(
-                "~", ".cache", "snowflake", "ocsp_response_validation_cache"
-            ),
-            "darwin": os.path.join(
-                "~", "Library", "Caches", "Snowflake", "ocsp_response_validation_cache"
-            ),
-            "windows": os.path.join(
-                "~",
-                "AppData",
-                "Local",
-                "Snowflake",
-                "Caches",
-                "ocsp_response_validation_cache",
-            ),
-        },
-    )
-except OSError:
-    # In case we run into some read/write permission error fall back onto
-    #  in memory caching
-    OCSP_RESPONSE_VALIDATION_CACHE: SFDictCache[
-        tuple[bytes, bytes, bytes],
-        OCSPResponseValidationResult,
-    ] = SFDictCache(
-        entry_lifetime=constants.DAY_IN_SECONDS,
-    )
+# use _get_ocsp_response_validation_cache instead of the module variable for lazy evaluation
+OCSP_RESPONSE_VALIDATION_CACHE: SFDictFileCache[
+    tuple[bytes, bytes, bytes], OCSPResponseValidationResult
+] | SFDictCache[tuple[bytes, bytes, bytes], OCSPResponseValidationResult] | None = None
```

Comment:
Drop the `None` here and make it be the return value of `_get_ocsp_response_validation_cache`
```suggestion
] | SFDictCache[tuple[bytes, bytes, bytes], OCSPResponseValidationResult]  = _get_ocsp_response_validation_cache()
```
This way you can drop the types and push them onto the function's return type-hint as well.

Reply:
I want this OCSP_RESPONSE_VALIDATION_CACHE to be lazy instantiated. calling the function conflicts my purpose...

Reply:
I have revert the change to lazy initialization, by simply moving the import into the if condition in `ssl_wrap_socket`:
```python
    if FEATURE_OCSP_MODE != OCSPMode.INSECURE:
        from .ocsp_asn1crypto import SnowflakeOCSPAsn1Crypto as SFOCSP
```